optimum/modeling_base.py

```diff
@@ -370,3 +371,10 @@ def from_pretrained(
             trust_remote_code=trust_remote_code,
             **kwargs,
         )
+
+    @classmethod
+    def _auto_model_to_task(cls, auto_model_class):
+        """
+        Get the task corresponding to a class (for example AutoModelForXXX in transformers).
+        """
+        return TasksManager.infer_task_from_model(auto_model_class)
```

Comment:
How different is this classmethod different from directly calling `TasksManager.infer_task_from_model(auto_model_class)`?